docs/en/deployment/third-party-tool-btpanel.md

```
@@ -0,0 +1,58 @@
+   ![install-Apollo](https://github.com/user-attachments/assets/e80dc7c0-e469-49b0-bd7f-225df36cdcab)
+
+4. configure basic information such as the domain name, ports to complete the installation
+
+   Note:
+   The domain name is optional, if the domain name is filled, it can be managed through [Website]--> [Proxy Project], and you do not need to check [Allow external access] after filling in the domain name, otherwise you need to check it before you can access it through the port
+
+   ![addApollo](https://github.com/user-attachments/assets/9623c6d9-d0e5-4f87-adea-e5a95133cf3c)
+
+5. After installation, enter the domain name or IP+ port set in the previous step in the browser to access.
+
+  - Name: application name, default `Apollo-characters`
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)